secrets-provider-chart/Chart.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v1
+appVersion: "1.0.0"
+description: A Helm chart for Kubernetes
```

Comment:
sound too generic for me. what about A helm chart for deploying CyberArk Secrets Provider for Kubernetes

Reply:
whoops good batch you're absolutely right